src/devtools/store.js

```diff
@@ -625,7 +625,9 @@ export default class Store extends EventEmitter {
     }
 
     const addedElementIDs: Array<number> = [];
-    const removedElementIDs: Array<number> = [];
+    // This is a mapping of removed ID -> parent ID:
+    const removedElementIDs: Map<number, number> = new Map();
+    // We'll use the parent ID to adjust selection if it gets deleted.
```

Comment:
Why is this comment below the line it's referring to? 🙃

Reply:
I wanted to keep the colon at the end of the first line but then it looked weird if I put a second comment line right after that. I dunno. Is it too weird?

Reply:
It reads kind of confusingly to me.

Almost like a with in the order sentence the words